tests/compare_dyldinfos.rs

```diff
@@ -12,6 +12,8 @@ fn get_realpath(cmd: &str) -> String {
 pub fn compare(args: Vec<&str>) {
     let apple = process::Command::new("/usr/bin/xcrun")
         .arg("dyldinfo")
+        .arg("-arch")
+        .arg("x86_64")
```

Comment:
This won’t run on a Mojave or Catalina system right ? There won’t be an arch flag ? (Just wondering if the example will work on non Big Sur systems or only Big Sur systems)

Reply:
It looks like `-arch` was a flag back in Xcode 11.3 https://github.com/keith/xcode-man-pages/blob/db47aee1c9b7493f6af64bfdf467b68706ce60b8/docs/dyldinfo.1.html so I think it would still work on those machines. Which makes sense to me given you would have needed it for i386 + x86_64 on macOS as well. Unless you're thinking that passing this would fail on non multi-arch binaries? (quick test on big sur still on a binary with just x86_64 works as expected)

Reply:
Awesome thank you for checking !